samples/net/sockets/echo_server/Kconfig

```diff
@@ -58,4 +58,12 @@ config NET_SAMPLE_PSK_HEADER_FILE
 	  Name of a header file containing a
 	  pre-shared key.
 
+config NET_SAMPLE_CERTS_WITH_SC
+	bool "Signed certificates"
+	default n
```

Comment:
`default n` can be removed as the setting is not enabled by default

Reply:
Done.